sdk/go/pulumi/provider/empty.s

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This empty .s file enables the use of go:linkname to make certain unexported functions from
```

Comment:
Interesting--are you sure this file is necessary? I think I've used `//go:linkname` w/o something like this in the past.

Reply:
Without it there are compilation errors about missing function bodies.

Reply:
Weird. SGTM, then 👍